Documentation outlining an example of using the OAI-PMH view that ships with Archipelago

theMusician opened the pull request:
The OAI-PMH setup that comes in Archipelago 1.4.0 is great. I had some struggles figuring out how exactly to leverage it to show a different collection than the demo one. 

The existing documentation, https://docs.archipelago.nyc/1.4.0/open_api/, does a great job explaining how it is all setup, but I needed some more assistance, out of that, is born this documentation. 

I nested the new page underneath the Metadata API module in the docs navigation.

Please let me know what needs to be changed for consideration in the documentation. 

Thank you

WWU-MaxBronsema committed:
Documentation for how to update Drupal core

theMusician committed:
Update docs/archipelago-deployment-live-update_drupal.md

Co-authored-by: Albert Min <albert.min@gmail.com>

theMusician committed:
Update docs/archipelago-deployment-live-update_drupal.md

Co-authored-by: Albert Min <albert.min@gmail.com>

theMusician committed:
Update docs/archipelago-deployment-live-update_drupal.md

Co-authored-by: Albert Min <albert.min@gmail.com>

theMusician committed:
Update docs/archipelago-deployment-live-update_drupal.md

Co-authored-by: Albert Min <albert.min@gmail.com>

WWU-MaxBronsema committed:
Updating file name as suggested and reverting table of contents.

WWU-MaxBronsema committed:
Renamed the file drupal_core_update.md

WWU-MaxBronsema committed:
First draft taking my written notes and converting them to markdown.

WWU-MaxBronsema committed:
Added screenshot and clarified role permissions.

WWU-MaxBronsema committed:
Improved formatting.

WWU-MaxBronsema committed:
Adding OAI example to documentation nav.

WWU-MaxBronsema committed:
The child page requires reformatting the YAML to meet mkdocks requirements.

alliomeria commented:
Thanks for submitting this @WWU-MaxBronsema ! We really appreciate your time preparing @DiegoPino 's Archipelago Slack and email discussion notes + your own experiences, into documentation updates! 

We will review and get back to you with feedback about these docs later this week. Thanks again!

alliomeria committed:
Update open_api_example.md

Update title and add intro text note --first pass of updates for this documentation

alliomeria commented:
Ok @theMusician , we'll be merging this now but will continue to make additional updates in the future. We will also be updating the main Metadata API docs again soon to account for some planned changes to the module. Thanks again for taking the time to write up example use case notes!

theMusician commented:
Thank you @alliomeria. I'll try to monitor changes to these pages in the future. Thank you for your work on Archipelago.
